Cargo.toml

```diff
@@ -69,7 +69,7 @@ wasm-bindgen-futures = "0.4"
 rand = "0.8"
 serde_json = "1.0"
 pyo3 = "0.20"
-quantum-sparse-sim = { git = "https://github.com/qir-alliance/qir-runner", rev = "54f2ffc097861146a2cc6595edaaffa7e2fdbf15", default-features = false }
```

Comment:
I'm guessing the removal of `default-features = false` was deliberate (doesn't appear to be any optional features on it).

Reply:
Yup, intentional. As of https://github.com/qir-alliance/qir-runner/pull/164 there is no longer any default/optional dependencies in the simulator, so while it has no effect it's clearer to remove it.